myblog/views.py

```diff
@@ -0,0 +1,42 @@
+from django.http import HttpResponse, JsonResponse
+
+from myblog.models import Post, PostView
+
+
+def getPosts(request):
+    posts_to_return = list();  # Make an empty list
+
+    m = 0
+    while True:
+        m = m + 1
+        if m >= Post.objects.count():
+            break;
+
+        l = Post.objects.all()[m];
+
+        posts_to_return.append({
+            'id': l.id,
+            'author': l.author,
+            'title': l.title,
+            'content': l.content,
+            'count': len(PostView.objects.filter(forWhatPost=l.id))
```

Comment:
Should be able to get this from the Post object by asking for the count of the reverse relation.